source/_components/sensor.github.markdown

```diff
@@ -62,6 +62,10 @@ access_token:
   description: Your GitHub Access Token
   required: false
   type: string
+server_url:
+  description: If you are using a GitHub Enterprise server, add it's URL here. For example, `https://mygithubserver.com`
+  required: false
```

Comment:
Yes, it defaults to `https://github.com` as the default via the library

Reply:
"If you use GitHub Enterprise put the url here."
default: https://github.com

Reply:
This URL should be blank / not added still since the library handles the defaults. If someone set `https://github.com` as the `server_url` it may cause issues or would have no effect